starter_code/queries.md

```diff
@@ -4,80 +4,102 @@
 
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+Filter : {"name":"Babelgum"}
+Project : {"name" : 1, _id :0}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+Filter : {"number_of_employees":{$gt:5000}}
+Sort : { "number_of_employees":-1}
+limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+Filter : {$and :[{"founded_year":{$gte: 2005}},{"founded_year":{$gte: 2000}}]}
+Project : {"name":1,"founded_year":1, _id:0}
 
 ### 4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
 
-<!-- Your Code Goes Here -->
+Filter : {"ipo.valuation_amount":{$gt:100000}}
+Project: {"name":1,"ipo":1, _id:0}
```

Comment:
Ici, il te manque la partie "founded before 2010" ;)
Ca ressemblerait à ça :
`{$and :[{"ipo.valuation_amount":{$gt:100000000}},{"founded_year":{$lt: 2010}}]}`